cli/deploy-torchserve.sh

```diff
@@ -0,0 +1,95 @@
+BASE_PATH=endpoints/online/custom-container
+AML_MODEL_NAME=torchserve-densenet161
+AZUREML_MODEL_DIR=azureml-models/$AML_MODEL_NAME/1
+MODEL_BASE_PATH=/var/azureml-app/$AZUREML_MODEL_DIR
+ENDPOINT_NAME=torchserve-endpoint
+
+# Download model and config file
+echo "Downling model and config file..."
+mkdir $BASE_PATH/torchserve
+wget --progress=dot:mega https://aka.ms/torchserve-densenet161 -P $BASE_PATH/torchserve
+wget --progress=dot:mega https://aka.ms/torchserve-config -P $BASE_PATH/torchserve
+
+# Get name of workspace ACR, build image
+WORKSPACE=$(az config get --query "defaults[?name == 'workspace'].value" -o tsv)
+ACR_NAME=$(az ml workspace show -w $WORKSPACE --query container_registry -o tsv | cut -d'/' -f9-)
+
+if [[ $ACR_NAME == "" ]]
+then
+    echo "ACR login failed, exiting"
+    exit 1
+fi
+
+az acr login -n $ACR_NAME
+IMAGE_TAG=${ACR_NAME}.azurecr.io/torchserve:8080
+az acr build $BASE_PATH/ -f $BASE_PATH/torchserve.dockerfile -t $IMAGE_TAG -r $ACR_NAME
+
+# Run image locally for testing
+docker run --rm -d -p 8080:8080 --name torchserve-test \
+    -e MODEL_BASE_PATH=$MODEL_BASE_PATH \
+    -v $PWD/$BASE_PATH/torchserve:$MODEL_BASE_PATH/torchserve $IMAGE_TAG
+
+sleep 10
+
+# Check Torchserve health
+echo "Checking Torchserve health..."
+curl http://localhost:8080/ping
+
+# Download test image
+echo "Downloading test image..."
+curl -O https://raw.githubusercontent.com/pytorch/serve/master/docs/images/kitten_small.jpg
```

Comment:
Better to put this in the public-models blob store?

Reply:
Good point. It would be better we manage this test image.